01-start/03-begin.html.md

```diff
@@ -27,12 +27,13 @@ Before we get hacking away on content, we have to create our project and set it
 ``` bash
 mkdir my-new-website
 cd my-new-website
+docpad --offline init
```

Comment:
> A secondary purpose is to minimize the number of things and concepts the user is presented with during the initial tutorial, so as to reduce cognitive load and avoid the user feeling that things are too complex. I think it's best to, if possible, avoid any mention of skeletons until another tutorial.

That's fair.



> The primary use is to work around the DocPad bug that causes the whole process to fail if you're initializing in a directory that's already part of a Git repo (docpad issue docpad/docpad#1030). I'd be happy to sit down and help you build a test case to demonstrate this (and whatever bits of new test framework we need to support such a test case); grab me on the Bevry Slack if you'd lik

That needs to be addressed. Surprisingly that the `--offline` flag fixes it. Let's continue on https://github.com/docpad/docpad/issues/1030